### PR TITLE
Feat: add zap log print

### PR DESCRIPTION
**Pull request by fengshunli**

### CODE REVIEW

File: internal/build/build.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build debug
```
Comment (Cyber-SiKu) [inline internal/build/build.go:1]: no need

File: internal/build/release.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//go:build !debug
```
Comment (Cyber-SiKu) [inline internal/build/release.go:1]: ditto

File: pkg/log/zaplog/log.go [hunk 3]
```diff
@@ -25,55 +25,115 @@
...
+	enc.AppendString(t.Format("curveadm" + " 2006/01/02 15:04:05"))
+}
+
+func getWriteSyncer() zapcore.WriteSyncer {
+	lumberJackLogger := &lumberjack.Logger{
+		Filename:   logpath, // 日志文件的位置
+		MaxSize:    10,      // 在进行切割之前，日志文件的最大大小（以MB为单位）
+		MaxBackups: 200,     // 保留旧文件的最大个数
+		MaxAge:     30,      // 保留旧文件的最大天数
+		Compress:   true,    // 是否压缩/归档旧文件
```
Comment (Cyber-SiKu) [inline pkg/log/zaplog/log.go]: English

File: cli/cli/cli.go [hunk 4]
```diff
@@ -131,15 +132,11 @@ func (curveadm *CurveAdm) init() error {
...
-	now := time.Now().Format("2006-01-02_15-04-05")
-	logpath := fmt.Sprintf("%s/curveadm-%s.log", curveadm.logDir, now)
-	if err := log.Init(config.GetLogLevel(), logpath); err != nil {
-		return errno.ERR_INIT_LOGGER_FAILED.E(err)
-	} else {
-		log.Info("Init logger success",
-			log.Field("LogPath", logpath),
-			log.Field("LogLevel", config.GetLogLevel()))
-	}
+	logpath := fmt.Sprintf("%s/curveadm.log", curveadm.logDir)
```
Comment (Cyber-SiKu) [inline cli/cli/cli.go:135]: Why is the date-related information removed from the log name?

File: cli/cli/cli.go [hunk 5]
```diff
@@ -148,17 +145,17 @@ func (curveadm *CurveAdm) init() error {
 	dbpath := fmt.Sprintf("%s/curveadm.db", curveadm.dataDir)
 	s, err := storage.NewStorage(dbpath)
 	if err != nil {
-		log.Error("Init SQLite database failed",
-			log.Field("Error", err))
+		zaplog.LOG.Error("Init SQLite database failed",
```
Comment (Cyber-SiKu) [inline cli/cli/cli.go:148]: You can avoid directly exposing the variable LOG by adding functions in zaplog.go, which can also be used in the same way as the original, such as zaplog.Info(), zaplog.Error() and other functions.

File: cli/cli/cli.go [hunk 6]
```diff
@@ -41,7 +42,7 @@ import (
 	tui "github.com/opencurve/curveadm/internal/tui/common"
 	"github.com/opencurve/curveadm/internal/utils"
 	cliutil "github.com/opencurve/curveadm/internal/utils"
-	log "github.com/opencurve/curveadm/pkg/log/glg"
+	"github.com/opencurve/curveadm/pkg/log/zaplog"
```
Comment (Cyber-SiKu) [inline cli/cli/cli.go:45]:
```suggestion
	log "github.com/opencurve/curveadm/pkg/log/zaplog"
```
This keeps your changes small.

File: pkg/log/zaplog/log.go (hunk 3 above)
Comment (fengshunli) [reply pkg/log/zaplog/log.go]: done

File: cli/cli/cli.go (hunk 4 above)
Comment (fengshunli) [reply cli/cli/cli.go:135]: The content of the same file reaches the specified number to be compressed. Passing the date will result in too many small files and cannot be cleared

File: internal/build/build.go (hunk 1 above)
Comment (fengshunli) [reply internal/build/build.go:1]: go fmt

File: internal/build/release.go (hunk 2 above)
Comment (fengshunli) [reply internal/build/release.go:1]: go fmt

File: cli/cli/cli.go (hunk 5 above)
Comment (fengshunli) [reply cli/cli/cli.go]: done

File: cli/cli/cli.go (hunk 6 above)
Comment (fengshunli) [reply cli/cli/cli.go:45]: done

**Comment by Cyber-SiKu** [comment]
There are some conflicts that can be resolved

**Comment by fengshunli** [comment]
see https://github.com/opencurve/curveadm/pull/233

**Pull request closed without merging**